.github/workflows/close-stale-issues.yaml

```diff
@@ -0,0 +1,29 @@
+---
+jobs:
+  stale:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: 'actions/stale@v9'
+      with:
+        days-before-close: '15'
+        days-before-stale: '45'
+        exempt-issue-labels: 'stale-exempt'
+        exempt-pr-labels: 'stale-exempt'
+        operations-per-run: '50' # this is to avoid Github API rate limiting.
+        stale-issue-message: >
+          This issue is stale because it has been open 45 days with no activity.
+          Remove the `stale` label or comment -
+          otherwise this will be closed in 15 days.
+        stale-pr-message: >
+          This issue is stale because it has been open 45 days with no activity.
+          Remove the `stale` label, comment or push a commit -
+          otherwise this will be closed in 15 days.
+name: 'Close stale issues and PRs'
+'on':
+  schedule:
+  - cron: '*/5 0 * * *' # cron scheduler syntax
+permissions:
+  # contents: write # this is only necessary if we use the delete-branch option.
```

Comment:
Nit: do we need this comment? I was confused at first because it looked like something is commented out